File: GreenScreen/app/src/main/java/com/example/greenscreen/vehicles/Bus.java
package com.example.greenscreen.vehicles;

import android.os.Bundle;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

import com.example.greenscreen.R;

public class Bus extends AppCompatActivity {
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_bus);
        TextView CO2 = (TextView) findViewById(R.id.text_view);
        CO2.setText("CO2 emissions: ");
    }
}

File: GreenScreen/app/src/main/java/com/example/greenscreen/vehicles/Bus.kt
package com.example.greenscreen.vehicles

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.greenscreen.R
import kotlinx.android.synthetic.main.activity_walk.*

class Bus : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_bus)
        text_view.setText("CO2 emissions: ")
    }
}